cmd/influxd/launcher/cmd.go

```diff
@@ -182,11 +186,13 @@ func newOpts(viper *viper.Viper) *InfluxdOpts {
 		BoltPath:   filepath.Join(dir, bolt.DefaultFilename),
 		EnginePath: filepath.Join(dir, "engine"),
 
-		HttpBindAddress:      ":8086",
-		HttpTLSMinVersion:    "1.2",
-		HttpTLSStrictCiphers: false,
-		SessionLength:        60, // 60 minutes
-		SessionRenewDisabled: false,
+		HttpBindAddress:       ":8086",
+		HttpReadHeaderTimeout: 10 * time.Second,
+		HttpIdleTimeout:       2 * time.Minute,
```

Comment:
Self-review: need to check what the default connection lifetime is when keep-alive is enabled.

Reply:
Answer: looks like there used to be a default of 3 minutes, but it was removed at some point. I'll update this default to match the old one.